ui/src/version/deployment/components/EndpointDeployment.js

```diff
@@ -79,12 +79,9 @@ const targetRequestStatus = currentStatus => {
 };
 
 const isRequestConfigured = request => {
-  if (!request.transformer || !request.transformer.enabled) {
-    return false;
-  }
-
   if (
-    request.transformer.transformer_type !== "standard" &&
+    request.transformer &&
```

Comment:
what if user use `standard` transformer there will be no image parameter that will be passed right?